GoogleForCreators/web-stories-wp
# Explore Templates: Restore Story Creation From Template

## Context

Unable to create stories from templates! 

## Summary

in #10452 elements were moved and now we need to register types at the app root. this wasn't added to the dashboard. causing issues for creating stories from templates.

Should eliminate need for #10967 

## Relevant Technical Choices

Since I was in `dashboard.js` I removed the setAppElement there - following this comment from a little bit ago so that that redundancy is gone https://github.com/GoogleForCreators/web-stories-wp/pull/10469#discussion_r800825893 

## To-do



## User-facing changes

Functionality to create stories from templates should be restored. 

## Testing Instructions

See that e2e tests for creating story from template are passing.
Make story from template using 'use template' button in both explore templates and in the template modal. 





This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10966
